example/applications/typescript/README.md

```diff
@@ -0,0 +1,10 @@
+# Typescript application
+
+Full documentation about it [here](https://mochajs.org/#-require-module-r-module)
+
+Tests here are kept next to their code (not in a separate dir).
+
+## npm scripts
```

Comment:
Can't say I see why this section is included in "README"s as it's self-apparent from "package.json". Suggest instead to convert to set of shell commands used to show how to use example.